scripts/superdesk/editor2/views/block-text.html

```diff
@@ -20,7 +20,7 @@
     </code>
 
     <ul class="dropdown-menu">
-        <li ng-repeat="word in suggestions"><button ng-click="replace(word)">{{ :: word }}</button></li>
+        <li ng-repeat="word in suggestions"><button ng-click="replace(word.key)">{{ :: word.value }}</button></li>
```

Comment:
Hi @lnogues , there is a change impact in superdesk\editor2\editor.js, 
- the below code need to be adjusted for capital word suggestion,

spellcheck.suggest(node.textContent).then(function(suggestions) {
                            if (scope.sentenceWord) {
                                ```suggestions.push(scope.replaceWord[0].toUpperCase() + scope.replaceWord.slice(1));```
                                scope.suggestions = _.unique(suggestions);
                            } else {

- use _.uniq instead, as _.unique alias is removed in favor of _.uniq in ver. 4. https://github.com/lodash/lodash/wiki/Changelog